ClipboardZanager/Sources/ClipboardZanager.Core.Desktop/Services/DataService.cs

```diff
@@ -306,14 +308,48 @@ internal List<DataIdentifier> GetDataIdentifiers(string[] formats)
             Requires.NotNull(foregroundWindow, nameof(foregroundWindow));
 
             var shouldSynchronize = true;
-            
+
+            BitmapImage icon = foregroundWindow.Icon;
+
+            if (icon == null)
+            {
+                icon = new BitmapImage();
+                StringBuilder className = new StringBuilder(256);
+                if (ServiceLocator.GetService<WindowsService>().GetClassName(NativeMethods.GetForegroundWindow(), className, 256) > 0)
```

Comment:
IS there another way to detect the desktop? Like "no foreground window handle", "special title", "explore.exe if foreground but there is not icon"...etc ?